src/mysql.c

```diff
@@ -973,6 +977,31 @@ static int mysql_read (user_data_t *ud)
 				counter_submit ("mysql_innodb_rows", "read", val, db);
 			else if (strcmp (key, "Innodb_rows_updated") == 0)
 				counter_submit ("mysql_innodb_rows", "updated", val, db);
+
+ 			/* checkpoints */
+ 			else if (strcmp (key, "Innodb_checkpoint_age") == 0)
+ 				gauge_submit ("mysql_innodb_checkpoint", "age", val, db);
```

Comment:
The type *mysql_innodb_checkpoint* does not exist. What's the unit of this metric? Seconds?